css/index.css

```diff
@@ -1,3 +1,158 @@
 /* Example of @import use
 This imports the `reset.css` file */
 @import './reset.css';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  list-style: none;
+  text-decoration: none;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
```

Comment:
Please note that we shouldn't have any CSS code in this file. This file is only meant to import other cs codes. Please create another fie called main.css where you can transfer the CSS in this file and import that here